### ISSUE TITLE
ConnectionPool polite close() call?

### ISSUE THREAD

**Issue by smferguson** [wbolster/happybase#120]
Hey guys,
We're beginning use of happybase and will be rolling it into prod in the coming weeks.
I'm curious why the connectionpool doesn't have a polite close() call.
I have code locally that loops over the pool and tries to close the connections, but I feel like there may be context (history) I'm missing.
Thanks,
Scott

**Comment by wbolster** [comment]
No, there isn't any history or other context. A simple loop should do the trick indeed, perhaps it makes sense to have it in Happybase itself.

**Comment by smferguson** [comment]
Cool. I'll submit a PR. Thanks for the quick reply.
